WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabViewController.swift

```diff
@@ -96,9 +96,9 @@ class ReaderTabViewController: UIViewController {
     }
 
     func presentDiscoverTab() {
-        viewModel.shouldShowCommentSpotlight = true
         viewModel.fetchReaderMenu()
         viewModel.showTab(at: ReaderTabConstants.discoverIndex)
+        displayNotice(title: NSLocalizedString("Comment to start making connections.", comment: "Hint for users to grow their audience by commenting on other blogs."))
```

Comment:
All the code related to `shouldShowCommentSpotlight` was doing was showing this notice after switching to the Reader tab. There might've been some sort of a spotlight (where?) at some point but it's long gone. Here's the flow:

<img width="240" alt="Screenshot 2024-08-29 at 4 43 48 PM" src="https://github.com/user-attachments/assets/050a179e-7125-4416-93f3-354ae6a1bd0d">  <img width="240" alt="Screenshot 2024-08-29 at 4 48 53 PM" src="https://github.com/user-attachments/assets/a4a51d04-8e79-4630-9f6f-adec6cc3ab8f">

Since the notices is a global system, of course, there is much easier way to implement it.

(nit) yeah I didn't add a new l10n with a namespace 😅

Reply:
I'm not sure this toast adds much value for the user – WDYT?

Reply:
It's one line of code now, so it doesn't bother me. I don't think it adds any value because it duplicates the message from the original cell in the Insights.